Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1885,6 +1885,41 @@ def get_path_data():
     return_results(results)
 
 
+def github_releases_list_command():
+    """
+    Gets releases data of given repository in given organization.
+    Returns:
+        CommandResults data.
+    """
+    args: Dict[str, Any] = demisto.args()
+
+    repo: str = args.get('repository') or REPOSITORY
+    organization: str = args.get('organization') or USER
+    limit = int(args.get('limit', 30))
```

Comment:
please use our standard for commands with pagination:
https://xsoar.pan.dev/docs/integrations/code-conventions#pagination-in-integration-commands